frame/uniques/src/lib.rs

```diff
@@ -192,62 +192,95 @@ pub mod pallet {
 	#[pallet::generate_deposit(pub(super) fn deposit_event)]
 	pub enum Event<T: Config<I>, I: 'static = ()> {
 		/// An asset class was created. \[ class, creator, owner \]
```

Comment:
This is now redundant and should be removed whenever we use a struct like variant.

```suggestion
		/// An asset class was created.
```

Reply:
Okay thanks. Should I do this change for all other structs too?

Reply:
Yes.